clients/docs/includes/configs/cloud/librdkafka.config

```diff
@@ -4,3 +4,6 @@ security.protocol=SASL_SSL
 sasl.mechanisms=PLAIN
 sasl.username={{ CLUSTER_API_KEY }}
 sasl.password={{ CLUSTER_API_SECRET }}
+
+# Increase session timeout for higher availability (45s is the default for Apache Kafka 3.0 and above)
```

Comment:
For the comments in here for librdkafka, can you verify which version the default change was made in?

Reply:
ah, it's 1.7 for Librdkafka. fixing it.